veles/secrets/common/pair/pair.go

```diff
@@ -53,17 +54,22 @@ type Detector struct {
 	FindA, FindB func(data []byte) []*Match
 	// Returns a veles.Secret from a Pair.
```

Comment:
"from a Pair and the underlying data that was searched" - same for FromPartialPair.

Actually, I wonder if we could just add the `data []byte` directly as a field to Match. It's a pointer so we wouldn't be storing any extra data and that way we don't have to pass it in to each function call separately.